doc/md/writing-motoko/modules-and-imports.md

```diff
@@ -1,23 +1,29 @@
+---
+sidebar_position: 16
+---
+
 # Modules and imports
 
-This section provides examples of different scenarios for using the `module` and `import` keywords.
+## Overview
 
-To illustrate how these keywords are used, let’s step through some sample code.
+The design of Motoko strives to minimize built-in types and operations. Instead of built-in types, Motoko provides a base library of modules to handle many kinds of common operations and make the language feel complete. This base library is still evolving with modules that support core features, and the base library APIs are subject to change over time to varying degrees. You should note, in particular, that the size and number of modules and functions included in the base library is likely to increase dramatically. Updates to the base library modules might introduce breaking changes that require you to update your programs to remain compatible. Breaking changes are communicated through the [Motoko migration guides](../migration-guides/overview.md).
+
+This section provides examples of different scenarios for using the `module` and `import` keywords.
 
-## Importing from the Motoko base library
+## Importing from the base library
 
-One of the most common import scenarios is one that you see illustrated in the examples in this guide, in the Motoko projects in the examples repository, and in the tutorials involves importing modules from the Motoko base library. Importing modules from the base library enables you to re-use the values, functions and types defined in those modules rather than writing similar ones from scratch.
+You can find source code and documentation for Motoko base modules in the [motoko-base](https://github.com/dfinity/motoko-base) open source repository. There are instructions in the repository for generating a local copy of the current documentation for the Motoko base library.
```

Comment:
Shouldn't we first point them the base library documentation on the portal (or here in doc/md/base)?